src/platforms/NativeImagePicker.web.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
```suggestion
```

Is this needed? I removed it locally and TS transpiles correctly without errors

Reply:
This was to suit my lint setup, but it can be removed.